plugins/cdecl.py

```diff
@@ -4,4 +4,4 @@
 @hook.command
 def cdecl(inp):
 	'''.cdecl <expr> -- translate between C declarations and English, using cdecl.org'''
-	return http.get("http://cdecl.org/query.php", q=inp)
+	return http.get("https://xwd733f66f.execute-api.us-west-1.amazonaws.com/prod/cdecl_backend", q=inp)
```

Comment:
Maybe grep for this from the html on cdecl.org? It might make it a little more robust.